pages/linux/gcov.md

```diff
@@ -0,0 +1,29 @@
+# gcov
+
+> Code coverage analysis and profiling tool. Discover untested parts of your program.
+> Get copy of source code annotated with execution frequencies of code segments.
```

Comment:
Perhaps this will read better?
```suggestion
> Code coverage analysis and profiling tool that discovers untested parts of a program.
> Also displays a copy of source code annotated with execution frequencies of code segments.
```